rebasehelper/utils.py

```diff
@@ -138,6 +142,101 @@ def cprint(cls, message, fg=None, bg=None, style=None):
         else:
             print(message)
 
+    @staticmethod
+    def color_is_light(rgb):
+        """Function to determine whether a color is dark or light.
+
+        Args:
+            rgb (tuple): RGB tuple with last element specifying number of RGB digits.
+
+        Returns:
+            bool: Whether color is light or dark.
+
+        """
+        brightness = 1 - (0.299 * rgb[0] + 0.587 * rgb[1] + 0.114 * rgb[2]) / (16 ** rgb[3] - 1)
```

Comment:
Why are you subtracting one?

Reply:
When calculating perceptive luminiscence, you are using the highest value which the bit-width can hold. For example for 2-digit RGB it would be ff = 255 so you have to divide by 255 whereas 16 ** 2 = 256. So you have to subtract if I am not mistaken

Reply:
Right, that makes sense. Thanks.